tests/data/abs_files/ftp/acc-phys/papers/9411/9411001.abs

```diff
@@ -4,7 +4,7 @@ arXiv:acc-phys/9411001
 From: Salman Habib <NdENv@u0-4VU8.fWz4.XrM>
 Date: Tue, 1 Nov 1994 18:57:48 GMT   (199kb)
 
-Title: Symplectic Computation of Lyapunov Exponents
+Title: {\gamma}-Cascade Symplectic Computation of Lyapunov arXiv:1701.00001
```

Comment:
Was this `\gamma` actually present in the original abs? it might interfere with other tests if not. In general, curious why the abs file was changed